src/header/button_for_search/button_for_search.js

```diff
@@ -0,0 +1,56 @@
+import React from "react";
```

Comment:
Please follow one name styles for all files. 

Preferable is to use CamelCase for files and components